android/src/main/kotlin/com/hypertrack/sdk/flutter/common/Serialization.kt

```diff
@@ -0,0 +1,172 @@
+package com.hypertrack.sdk.flutter.common
+
+import android.location.Location
+
+/**
+ * Platform-independent serialization code that converts HyperTrack data types
+ * to Map<String, T> or List<T> where T is any JSON-compatible type
+ */
```

Comment:
that's a javadoc as well, and will be connected to `serializeErrors` only.
I suggest creating a class that wraps the global methods, you can still use imports to skip the class name in the code.

Reply:
fixed